binderhub/templates/index.html

```diff
@@ -14,6 +14,7 @@
 {% block main %}
 <div id="main" class="container">
   <div class="row">
+    <h3 class="text-center" style="margin-top: -60px;">(beta)</h3>
```

Comment:
you should add another id or class here, and add css rules for that in index.css :)

Reply:
fineeeee :-)